aptos-move/framework/aptos-framework/sources/configs/execution_config.move

```diff
@@ -0,0 +1,37 @@
+/// Maintains the execution config for the blockchain. The config is stored in a
+/// Reconfiguration, and may be updated by root.
+module aptos_framework::execution_config {
+    use std::error;
+    use std::vector;
+
+    use aptos_framework::reconfiguration;
+    use aptos_framework::system_addresses;
+
+    friend aptos_framework::genesis;
+
+    struct ExecutionConfig has key {
+        config: vector<u8>,
+    }
+
+    /// The provided on chain config bytes are empty or invalid
+    const EINVALID_CONFIG: u64 = 1;
+
+    /// Publishes the ConsensusConfig config.
+    public(friend) fun initialize(aptos_framework: &signer, config: vector<u8>) {
```

Comment:
We still need the initialize function to be called from `genesis.initialize()` right?

Reply:
Ah, I see what you mean. You are right, this can be replaced by the `set` function call but would prefer to leave it as it is to be consistent with how we are initializing other things from genesis.

Reply:
If you leave it as is, you'll need an extra governance proposal just to call initialize. Let's avoid that